internal/plugins/linux/selinux.go

```diff
@@ -150,25 +151,31 @@ func (self *SELinuxPlugin) sELinuxActive() bool {
 		sllog.WithError(err).Debug("Unable to use SELinux.")
 		return false
 	}
+
 	if _, _, err = self.parseSestatusOutput(output); err == ErrSELinuxDisabled {
 		sllog.WithError(err).Debug("Unable to use SELinux.")
 	}
 	return err == nil
 }
 
-func (self *SELinuxPlugin) parseSemoduleOutput(output string) (result types.PluginInventoryDataset, err error) {
-	moduleRegex, err := regexp.Compile(`(\S+)\s+(\S+)`)
-	if err != nil {
-		return
-	}
+func (self *SELinuxPlugin) parseSemoduleOutput(output string) (result types.PluginInventoryDataset) {
+	// Capture "zero-or-more elements" of whitespaces and non-whitespaces for the optional version field
+	moduleRegex := regexp.MustCompile(`(\S+)\s*(\S*)`)
 	scanner := bufio.NewScanner(strings.NewReader(output))
+	const captureGroupsMinLen = 2
```

Comment:
is recommended to add the constants on the top of the file after the imports and all together there. At the least for the good practices that i know

Reply:
Yeah this was probably my fault as I added the constant in the function body for my suggestion [here](https://github.com/newrelic/infrastructure-agent/pull/1940#pullrequestreview-2388875817). It's probably better if left as a variable inside the function body, as it's only used there.